src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/goal/Goal.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal
+
+import java.time.Instant
+import java.time.LocalDate
+import java.util.UUID
+
+data class Goal(
+  val reference: UUID,
+  val title: String,
+  val reviewDate: LocalDate,
+  val type: GoalType,
+  val steps: List<Step>,
+  val status: GoalStatus,
+  val notes: String?,
+  val createdBy: String,
+  val createdDate: Instant,
```

Comment:
`createdDate` (and `lastUpdatedDate`) or `createdDateTime`? or `creaatedAt` / `lastUpdatedAt` ?
🤔 Naming is hard!

Reply:
Opted for the latter 👍